cypress/pageObjects/assertions.ts

```diff
@@ -0,0 +1,22 @@
+import { eq } from "cypress/types/lodash"
+
+class Assertions {
+
+    checkHomePageIsOpen() {
+        cy.url().should('include', 'https://qaintazugg.restaurant365.com/#/grid/AllTransactionsGrid/00000000-0000-0000-0000-000000000000')
```

Comment:
baseUrl can be changed so you need only to check `#/grid/AllTransactionsGrid/00000000-0000-0000-0000-000000000000`